docs/asciidoc/front_matter.adoc

```diff
@@ -15,6 +15,12 @@ Each implementation of DRS can choose its own id scheme, as long as it follows t
 * DRS v1 does NOT support semantics around multiple versions of an object. (For example, there’s no notion of “get latest version” or “list all versions”.) Individual implementation MAY choose an ID scheme that includes version hints.
 * DRS implementations MAY have more than one ID that maps to the same object.
 
+=== DRS URIs
+
+For convenience, including when passing content references to a WES server, we define a URI syntax for DRS-accessible content. Strings of the form `drs://<server>/<id>` mean _“you can fetch the content with DRS id `<id>` from the DRS server at `<server>` "_.
```

Comment:
```
DRS IDs are URL-safe text strings made up of alphanumeric characters and any of [.-_/]
```

When we discussed in Hinxton we were saying that `/` wasn't valid so checking if a server existed would be trivial. However, that contradicts the other documents here referenced above. I propose we do `drs-<server>://<id>` or other such to make it trivial to separate in the future.

Reply:
Good catch @zflamig -- I was remembering that we prohibited slashes in `BundleObject.name`, and got that confused with IDs. But I think it's actually fine. Someone pointed out to me, `drs://<id>` isn't a valid URI, but `drs:<id>` is (which I think is supported by https://tools.ietf.org/html/rfc3986, but I gave up after 20 pages). So we're covered whenever we want to do a global ID scheme.

Personally, I still think slashes in IDs is bad practice, since life is easier if you never have to escape things. But I don't feel strongly, since escaping works.